### ISSUE TITLE
JSX member expressions are reported as unused variables

### ISSUE THREAD

**Issue by fireairforce** [utooland/utoo-lint#1045]
## Version

`@utoo/lint@0.2.5`

## Minimal reproduction

```tsx
import { motion } from "framer-motion";

export function App() {
  return <motion.div>Utoo</motion.div>;
}
```

Run either rule in isolation:

```bash
utoo-lint --rules=@typescript-eslint/no-unused-vars fixture.tsx --format=json
utoo-lint --rules=no-unused-vars fixture.tsx --format=json
```

Both commands report the import on line 1:

```json
{
  "line": 1,
  "column": 10,
  "severity": "warning",
  "message": "'motion' is declared but never used."
}
```

The first command reports `@typescript-eslint/no-unused-vars`; the second reports `no-unused-vars`.

## Expected behavior

The `motion` identifier in the JSX member expression `<motion.div>` is a value reference, so neither rule should report the import.

## Impact

The packaged frontend config enables both unused-variable rules. Framer Motion commonly renders components through member expressions such as `<motion.div>`, so adopting the preset still produces false-positive warnings after the direct JSX tag fix in #1044.

## Environment

- macOS arm64
- Node.js 22.23.1

**Comment by fireairforce** [comment]
Reproduced during the @utoo/lint adoption in https://github.com/utooland/site/pull/21. The site keeps both unused-variable rules enabled and temporarily ignores only variables named `motion` via `varsIgnorePattern`, so unrelated unused variables are still reported.
